app/routes/administrative-units/administrative-unit/core-data/edit.js

```diff
@@ -14,6 +14,25 @@ export default class AdministrativeUnitsAdministrativeUnitCoreDataEditRoute exte
       contacts.pushObject(this.store.createRecord('contact-point'));
     }
 
+    let identifiers = await administrativeUnit.get('identifiers');
+    if (identifiers.length === 1) {
+      let idName;
+      if (identifiers.firstObject.idName === 'SharePoint identificator') {
+        idName = 'KBO nummer';
```

Comment:
Since this is used in multiple places, I would create a const / enum thing for it that can be reused. This seems very typo prone